chi_edge/cli.py

```diff
@@ -159,3 +167,65 @@ def bootstrap(
     return ansible.run(
         "setup_edge_dev.yml", host, group=device_type, host_vars=host_vars
     )
+
+
+@cli.command(help="Bake a balena image")
+@click.argument("image",
+    help=(
+        "Path to the raw disk image to configure. This should be a `.img` "
+        "file."
+    ),
+)
+@click.argument("device_uuid",
+    help=(
+        "The Doni device uuid to configure the image for."
+    ),
+)
+def bake(image, device_uuid):
```

Comment:
Re: Mike's question, it is always an option to support just outputting the config JSON file directly. Perhaps `--image` can be an optional flag that, when present, uses the config.json in the image as the baseline, but also copies it back to the image. Otherwise it just outputs a config.json file?